From: Yanhao Mo <yanhaocs@gmail.com>
Date: Tue, 26 Feb 2019 17:15:10 +0800
Subject: build-with-debug-info

---
 src/makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/makefile b/src/makefile
index 0f61c35..eb8b319 100644
--- a/src/makefile
+++ b/src/makefile
@@ -1,5 +1,5 @@
 SHELL=/bin/bash
-CFLAGS=--std=c99
+CFLAGS=--std=c99 -g
 EXECUTABLES = find msgmerge msgfmt install rm mkdir cp chmod valac
 CHECKEXECS := $(foreach exec,$(EXECUTABLES), $(if $(shell which $(exec)),,$(error No $(exec) found, install it)))
 
@@ -50,7 +50,7 @@ all: app-gtk app-console
 app-gtk:
 
 	#timeshift-gtk
-	valac -X -D'GETTEXT_PACKAGE="${app_name}"' \
+	valac -g -X -D'GETTEXT_PACKAGE="${app_name}"' \
 		--Xcc="-lm" --Xcc="-Os" ${symbols} \
 		Core/*.vala Gtk/*.vala Utility/*.vala Utility/Gtk/*.vala \
 		-o ${app_name}-gtk \
@@ -61,7 +61,7 @@ app-gtk:
 app-console:
 
 	#timeshift
-	valac -X -D'GETTEXT_PACKAGE="${app_name}"' \
+	valac -g -X -D'GETTEXT_PACKAGE="${app_name}"' \
 		--Xcc="-lm" --Xcc="-Os" ${symbols} \
 		Core/*.vala Utility/*.vala Utility/Gtk/*.vala Console/*.vala \
 		-o ${app_name} \
