From: Yanhao Mo <yanhaocs@gmail.com>
Date: Wed, 23 Nov 2022 16:58:12 -0500
Subject: Add keywords to timeshift-gtk.desktop

---
 src/timeshift-gtk.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/timeshift-gtk.desktop b/src/timeshift-gtk.desktop
index 90dfe88..1fc7977 100644
--- a/src/timeshift-gtk.desktop
+++ b/src/timeshift-gtk.desktop
@@ -29,3 +29,4 @@ Comment[ru]=Программа для восстановления систем
 X-KDE-StartupNotify=false
 Categories=System;
 X-GNOME-UsesNotifications=true
+Keywords=backup;btrfs;rsync;
