From: Yanhao Mo <yanhaocs@gmail.com>
Date: Wed, 23 Nov 2022 16:58:12 -0500
Subject: Add VALAFLAGS to valac arguments in src/makefile.

---
 src/makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/makefile b/src/makefile
index eb8b319..4c3a0e4 100644
--- a/src/makefile
+++ b/src/makefile
@@ -52,6 +52,7 @@ app-gtk:
 	#timeshift-gtk
 	valac -g -X -D'GETTEXT_PACKAGE="${app_name}"' \
 		--Xcc="-lm" --Xcc="-Os" ${symbols} \
+		$(VALAFLAGS) \
 		Core/*.vala Gtk/*.vala Utility/*.vala Utility/Gtk/*.vala \
 		-o ${app_name}-gtk \
 		--pkg glib-2.0 --pkg gio-unix-2.0 --pkg posix \
@@ -63,6 +64,7 @@ app-console:
 	#timeshift
 	valac -g -X -D'GETTEXT_PACKAGE="${app_name}"' \
 		--Xcc="-lm" --Xcc="-Os" ${symbols} \
+		$(VALAFLAGS) \
 		Core/*.vala Utility/*.vala Utility/Gtk/*.vala Console/*.vala \
 		-o ${app_name} \
 		--pkg glib-2.0 --pkg gio-unix-2.0 --pkg posix \
